src/components/InstallButton.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Dropdown, Link } from '@newrelic/gatsby-theme-newrelic';
+import { css } from '@emotion/react';
+
+const sampleItems = new Array(10).fill().map((_, i) => i + 1);
+
+const createMenuItems = (children, items) => {
```

Comment:
This could be simplified to 
```suggestion
const createMenuItems = (items) => items.map((i) => <Dropdown.MenuItem>{i}</Dropdown.MenuItem>)
```
I omitted `children` since it doesn't appear to be used.

Reply:
@aswanson-nr I only kept that in because we will be using `children` eventually but I can take it out for now 🚸